### PR TITLE
Setting logout permission on a application when server-endpoint allow…

### PR DESCRIPTION
**Pull request by Tsjerno**
Setting logout permission on a application when serverendpoint allows logout.

When not setting this permission, OpenIddict.Server.OpenIddictServerProviderValidateLogoutRequest will return false.

The user will get an error "invalid_request The specified 'post_logout_redirect_url' parameter is invalid."
While the parameter is valid, but a permission is missing, which cannot be set in the application UI of OrchardCore

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/ApplicationController.cs [hunk 1]
```diff
@@ -167,6 +167,17 @@ public async Task<IActionResult> Create(CreateOpenIdApplicationViewModel model,
                 Type = model.Type
             };
 
+            var openIdServerSettings = await GetServerSettingsAsync();
```
Comment (PinpointTownes) [inline src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/ApplicationController.cs]: While I agree this must be fixed, I don't think it's the right thing to do. Instead, we should consider adding a new checkbox to allow clients to use the logout endpoint.

**Comment by Tsjerno** [comment]
@PinpointTownes as suggested i've added a checkbox on the application pages.
 - The checkbox toggles visibility of the logout uris
 - Checkbox is only available when logout endpoint is active on server level

logout

**Comment by PinpointTownes** [comment]
@Tsjerno great, thanks!

File: src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/ApplicationController.cs (hunk 1 above)
Comment (PinpointTownes) [reply src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/ApplicationController.cs]: Do we still need this call?

Comment (Tsjerno) [reply src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/ApplicationController.cs]: Nope, removed it

**Pull request merged**